webroot/js/components/chat/chat.js

```diff
@@ -333,7 +333,7 @@ export default class Chat extends Component {
       // update document title if window blurred
       if (
         this.numMessagesSinceBlur &&
-        !this.props.messagesOnly &&
+        !this.props.readonly &&
         this.windowBlurred
       ) {
         this.updateDocumentTitle();
```

Comment:
Now that people might be opening this standalone chat in a browser, people will see the title of the page.  Right now it sets it to `undefined` and looks broken, since I'm guessing there's some missing value somewhere.  It didn't matter before because the chat embed wasn't really being opened in the browser, but now people will probably see that weird title. 
![image](https://user-images.githubusercontent.com/414923/127408321-63f9e50c-1247-47fd-b428-3c609b489e08.png)

Reply:
I was not expecting people to open the embed chat directly in the browser😅. But it should be fixed now.